appchain/src/tests.rs

```diff
@@ -648,7 +654,8 @@ fn test_submit_validator_sets_on_chain() {
 			2,
 			"https://rpc.testnet.near.org",
 			b"oct-test.testnet".to_vec(),
-			public,
+			public.clone()  ,
+			public.clone().into_account().encode() , 	// default value.
```

Comment:
Maybe you can use ```public.into_account().encode()```

Reply:
Remove **.clone()** and formatted the code I modified.